### ISSUE TITLE
[SF][A11y][Avatar]: [Avatar icon text does not meet minimum color contrast requirement ]

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents-react#8178]
### Describe the bug

The avatar icon text does not meet the minimum required color contrast ratio of 4.5:1 and has a ratio of 4.2:1

<img width="200" height="109" alt="Image" src="https://github.com/user-attachments/assets/cb14e38a-674c-40ee-ab3d-b55e21aa9044" />

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/docs/data-display-avatar--docs

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Avatar text should meet the minimum color contrast ratio of 4.5:1 against its background.

### UI5 Web Components for React Version

2.18.0

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
